Bock2.py

```diff
@@ -0,0 +1,259 @@
+        self.middle_middle_Frame = Frame(self.tk, background=background)
+        self.middle_left_Frame = Frame(self.tk, background=background)
+        self.middle_right_Frame = Frame(self.scrollable_frame, background=background)
+        self.bottom_middle_Frame = Frame(self.scrollable_frame, background=background)
+        self.bottom_left_Frame = Frame(self.scrollable_frame, background=background)
+        self.bottom_right_Frame = Frame(self.tk, background=background)
+        # self.topFrame.pack(side=TOP, fill=BOTH, expand=YES)
+        # self.bottomFrame.pack(side=BOTTOM, fill=BOTH, expand=YES)
+        self.top_middle_Frame.grid(row=0, column=1, sticky="new")
+        self.top_left_Frame.grid(row=0, column=0, sticky="nw")
+        self.top_right_Frame.grid(row=0, column=2, sticky="ne")
+        self.middle_middle_Frame.grid(row=1, column=1, sticky="nsew")
+        self.middle_left_Frame.grid(row=1, column=0, sticky="nsw")
+        self.middle_right_Frame.grid(row=1, column=2, sticky="nse")
+        self.bottom_middle_Frame.grid(row=2, column=1, sticky="sew")
+        self.bottom_left_Frame.grid(row=2, column=0, sticky="sw")
+        self.bottom_right_Frame.grid(row=2, column=2, sticky="se")
+
+        # fullscreen
+        self.toggle_fullscreen()  # start in fullscreen
+
+        self.change2()
+        # ctrl+f to toggle fullscreen
+        self.tk.bind("<Control-f>", self.toggle_fullscreen)
+        self.tk.bind("<Escape>", self.end_fullscreen)
+        self.tk.bind("<Control-a>", self.change1)
+        self.tk2.bind("<Control-a>", self.change2)
+        
+
+        self.greetings_frame = Greeting_Frame(self.top_middle_Frame)
+        self.greetings_frame.grid(row=0, column=1, sticky="n")
+
+        self.picture_frame = Picture_Frame(self.top_left_Frame)
+        self.picture_frame.grid(row=0, column=0, sticky="nsew")
+
+        # clock
+        self.clock_frame = Clock_frame(self.top_right_Frame)
+        self.clock_frame.grid(row=0, column=2, sticky="ne")
+        # weather
+        self.weather_frame = Weather_frame(self.top_right_Frame)
+        self.weather_frame.grid(row=1, column=2, sticky="ne")
+        
+
+    # this is called when ctrl+f is pressed, it'll toggle the full screen
+
+    def toggle_fullscreen(self, event=None):
+        self.state = not self.state  # Just toggling the boolean
+        self.tk.attributes("-fullscreen", self.state)
+        return "break"
+
+    # this is called if esc is pressed, this will make the screen smaller
+    def end_fullscreen(self, event=None):
+        self.state = False
+        self.tk.attributes("-fullscreen", False)
+        return "break"
+
+    def calender(self):
+        print('stuff needs to go here')
+
+    def change1(self, *args):
+        self.tk.withdraw()
+        self.tk2.deiconify()
+        return
+
+    def change2(self, *args):
+        self.tk2.withdraw()
+        self.tk.deiconify()
+        return 
```

Comment:
withdraw() hides
deiconify() shows

Reply:
I can try to mess with the position and transparency instead, but this does the job